cheatsheets/Security_Definitions_Cheat_Sheet.md

```diff
@@ -0,0 +1,29 @@
+# Security Definitions Cheat Sheet
+
+## Encoding vs Escaping
```

Comment:
I almost think we should throw in "sanitization" into this grouping as well. Thoughts?